addons/project/views/project_portal_templates.xml

```diff
@@ -316,7 +316,7 @@
                                     <div t-else=""><strong>Project:</strong> <a t-field="task.project_id"/></div>
                                     <div t-if="task.date_deadline"><strong>Deadline:</strong> <span t-field="task.date_deadline" t-options='{"widget": "date"}'/></div>
                                     <div t-if="task.milestone_id and task.allow_milestones"><strong>Milestone:</strong> <span t-field="task.milestone_id"/></div>
-                                    <div name="portal_my_task_planned_hours">
+                                    <div name="portal_my_task_planned_hours" t-if="allow_timesheets">
```

Comment:
Again, it is not defined in that module. 😕

Reply:
done :+1: